llvm/lib/Support/Signals.cpp

```diff
@@ -273,9 +273,19 @@ static bool printMarkupStackTrace(StringRef Argv0, void **StackTrace, int Depth,
 }
 
 // Include the platform-specific parts of this class.
-#ifdef LLVM_ON_UNIX
+#if defined(__wasi__)
+// WASI does not have signals.
```

Comment:
Move these stubs to WASI/Signals.inc for consistency with the other platform support files.